Earthfile

```diff
@@ -0,0 +1,42 @@
+VERSION 0.6
+
+equivs:
+  FROM ubuntu:bionic
+  RUN apt-get update
+  RUN apt-get install -y equivs lintian
+
+rpmbuild:
+  FROM almalinux:8
+  RUN dnf install -y rpmdevtools rpmlint
+
+RPM:
+  COMMAND
+  ARG package
+  FROM +rpmbuild
+  COPY $package /tmp/$package
+  WORKDIR /tmp/$package
+  RUN rpmbuild -ba ${package}.spec --define "_topdir /tmp/$package/rpm" --define "_sourcedir /tmp/$package"
+  RUN rpmlint /tmp/$package/rpm/RPMS/noarch/${package}*.rpm
+  SAVE ARTIFACT /tmp/$package/rpm/RPMS/noarch/${package}*.rpm AS LOCAL output/
```

Comment:
We'll want the source RPM to upload to the repositories as well. It's at `/tmp/$package/rpm/SRPMS/${package}*.rpm`. Running rpmlint on that as well may give you more feedback. You can lint both the binary and source packages in the same invocation.

Reply:
> We'll want the source RPM..

Equally true of the sourcedeb, good spotting.

Reply:
[089615d](https://github.com/ros-infrastructure/infra-variants/pull/1/commits/089615d2b94dc0424e113f9569b0aee24cde7ce9) adds the SRPM to both the saved archives and the rpmlint invocation. There are two warnings related to lacking `%build` and `%install` sections which I think we can ignore but I'm not sure if it's better to do so via config file or just letting them be since warnings do not currently give a non-zero exist status. I think I'd prefer to be strict on warnings but I'll defer to your experience.

Reply:
Strict on warnings seems fine. It would probably be easiest to just include the empty `%build` and `%install` rather than deal with a suppression file - that seems to be a more common practice.